src/transformation/visitors/language-extensions/table.ts

```diff
@@ -114,41 +49,40 @@ function transformTableDeleteExpression(
     return lua.createBooleanLiteral(true);
 }
 
+function transformWithTableArgument(context: TransformationContext, node: ts.CallExpression): lua.Expression[] {
+    if (ts.isPropertyAccessExpression(node.expression) || ts.isElementAccessExpression(node.expression)) {
+        return transformExpressionList(context, [node.expression.expression, ...node.arguments]);
+    }
+    // todo: report diagnostic?
+    return [lua.createNilLiteral(), ...transformExpressionList(context, node.arguments)];
+}
+
 function transformTableGetExpression(
     context: TransformationContext,
     node: ts.CallExpression,
-    extensionKind: extensions.ExtensionKind
+    extensionKind: ExtensionKind
 ): lua.Expression {
-    const args = node.arguments.slice();
-    if (
-        extensionKind === extensions.ExtensionKind.TableGetMethodType &&
-        (ts.isPropertyAccessExpression(node.expression) || ts.isElementAccessExpression(node.expression))
-    ) {
-        // In case of method (no table argument), push method owner to front of args list
-        args.unshift(node.expression.expression);
-    }
+    const args =
+        extensionKind === ExtensionKind.TableGetMethodType
+            ? transformWithTableArgument(context, node)
+            : transformExpressionList(context, node.arguments);
 
-    const [table, accessExpression] = transformExpressionList(context, args);
-    // arg0[arg1]
+    const [table, accessExpression] = args; // arg0[arg1]
```

Comment:
Comment is misplaced now, it applies to the next line

Reply:
Not yet resolved

Reply:
I somehow managed to fix every other comment in the file except that one.
Fixed now.